win32ss/user/ntuser/painting.c

```diff
@@ -675,6 +675,11 @@ IntInvalidateWindows(PWND Wnd, PREGION Rgn, ULONG Flags)
          {
              RgnType = IntGdiCombineRgn(Rgn, Rgn, RgnWindow, RGN_AND);
              REGION_Delete(RgnWindow);
+             /* Nothing to paint, just return */
```

Comment:
Hmh..... Shouldn't this check and return be done a little bit later? Like for example at old line 700 (new line 705) ?
I would like to hear what IKillBugs would have to say about this, before proceeding further with this PR.

Reply:
@HBelusca, The logic has been relocated as you suggested now. Please review and confirm. Thanks.

Reply:
Just to clarify @Doug-Lyons , my suggestion has to be verified that it really works and is still correct in view of the rest of the code, this is also why I asked IKillBug (in the jira) to check what he thinks about it.